pkgs/desktops/gnome/misc/gnome-extensions-cli/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, python3
```

Comment:
Okay, one last nit and this looks good to me

```suggestion
```

This arg isn't used anymore

Reply:
Fixed :)